vars/dockerBuildArgs.groovy

```diff
@@ -56,75 +101,46 @@ String call(Map config = [:]) {
       ret_str += " --build-arg CB0=" + current_time.get(Calendar.WEEK_OF_YEAR)
     }
     if (add_repos) {
-      if (env.REPOSITORY_URL) {
-        ret_str += ' --build-arg REPO_URL=' + env.REPOSITORY_URL
-      }
-      string repo_name = null
+      String repo_name = null
       if (stage_info['target'] == 'centos7') {
-        if ((repo_type == 'local') && (env.DAOS_STACK_EL_7_LOCAL_REPO)) {
-            repo_name = env.DAOS_STACK_EL_7_LOCAL_REPO
-        } else if ((repo_type == 'stable') &&
-                   (env.DAOS_STACK_EL_7_DOCKER_STABLE_REPO)) {
-          repo_name = env.DAOS_STACK_EL_7_DOCKER_STABLE_REPO
-        } else if ((repo_tpe == 'release') &&
-                   (env.DAOS_STACK_EL_7_DOCKER_RELEASE_REPO)) {
-          repo_name = env.DAOS_STACK_EL_7_DOCKER_RELEASE_REPO
-        } else if ((repo_type == 'dev') &&
-                   (env.DAOS_STACK_EL_7_DOCKER_DEV_REPO)) {
-          repo_name = env.DAOS_STACK_EL_7_DOCKER_DEV_REPO
-        }
+        repo_alias = 'EL_7'
+        repo_name = env["DAOS_STACK_${repo_alias}${repo_mod}_${repo_type}_REPO"]
         if (repo_name) {
           ret_str += ' --build-arg REPO_EL7=' + repo_name
         }
       }
       if (stage_info['target'] == 'centos8') {
-        if ((repo_type == 'local') && (env.DAOS_STACK_EL_8_LOCAL_REPO)) {
-            repo_name = env.DAOS_STACK_EL_8_LOCAL_REPO
-        } else if ((repo_type == 'stable') &&
-                   (env.DAOS_STACK_EL_8_DOCKER_STABLE_REPO)) {
-          repo_name = env.DAOS_STACK_EL_8_DOCKER_STABLE_REPO
-        } else if ((repo_tpe == 'release') &&
-                   (env.DAOS_STACK_EL_8_DOCKER_RELEASE_REPO)) {
-          repo_name = env.DAOS_STACK_EL_8_DOCKER_RELEASE_REPO
-        } else if ((repo_type == 'dev') &&
-                   (env.DAOS_STACK_EL_8_DOCKER_DEV_REPO)) {
-          repo_name = env.DAOS_STACK_EL_8_DOCKER_DEV_REPO
-        }
+        repo_alias = 'EL_8'
+        repo_name = env["DAOS_STACK_${repo_alias}${repo_mod}_${repo_type}_REPO"]
         if (repo_name) {
           ret_str += ' --build-arg REPO_EL8=' + repo_name
         }
       }
       if (stage_info['target'] == 'leap15') {
-        if (repo_type == 'local') {
-          if (env.DAOS_STACK_LEAP_15_LOCAL_REPO) {
-            ret_str += ' --build-arg REPO_LOCAL_LEAP15=' +
-                       env.DAOS_STACK_LEAP_15_LOCAL_REPO
-          }
-          if (env.DAOS_STACK_LEAP_15_GROUP_REPO) {
+        repo_alias = 'LEAP_15'
+        repo_name = env["DAOS_STACK_${repo_alias}${repo_mod}_${repo_type}_REPO"]
+        if (repo_name) {
+          ret_str += ' --build-arg REPO_EL7=' + repo_name
+        }
```

Comment:
Basically ditto for the above, but I doubt the --build-arg is supposed to be `REPO_EL7` here.

Reply:
Bug, will be fixed in next push.